src/app/components/feed/FeedPageContent.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { FormattedMessage, FormattedHTMLMessage } from 'react-intl';
+import { browserHistory } from 'react-router';
+import cx from 'classnames/bind';
+import AddIcon from '../../icons/add.svg';
+import FeedIcon from '../../icons/dynamic_feed.svg';
+import styles from './FeedPage.module.css';
+import Can from '../Can';
+import ButtonMain from '../cds/buttons-checkboxes-chips/ButtonMain';
+
+const FeedSection = ({
+  title, content,
+}) => (
+  <div className={cx(styles['feed-content-wrapper'], styles['feed-content'])}>
+    <h2 className="typography-subtitle2">
+      {title}
+    </h2>
+    <p className="typography-body1">
```

Comment:
@danielevalverde you don't need the className on `<p>` tags, they automatically get the correct css from global typography.css file